.github/workflows/py-unit-tests.yml

```diff
@@ -1,52 +1,18 @@
-# Workflow that runs frontend and python unit tests
-name: Run Unit Tests
-
-# Only run one workflow of the same group at a time.
-# There can be at most one running and one pending job in a concurrency group at any time.
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+# Workflow that runs python unit tests
+name: Run Python Unit Tests
 
+# * Always run on "main"
+# * Always run on PRs
 on:
   push:
     branches:
       - main
-    paths-ignore:
```

Comment:
Why are these being removed?

Reply:
I can these back if you think this workflow is too painful. (Takes 5 min).
My reasoning was that unit tests should just always be run but happy to revert this one.

Reply:
It was added as a measure to bring total workflow time down. E.g. a change to documents doesn't warrant any unit tests etc. Unless the ignored paths prevent testing or dependent workflows, I'd prefer to leave them as they were.

Reply:
Sure. Readding now

Reply:
Totally agree, on `main`/`push` all should run.

Reply:
Yeah see so the problem is the one we've been running into all the time.
This repo is setup so the unit tests for python are required so you cannot add paths-ignore to these jobs, otherwise it will never run and not let you merge anything.

This was setup so it doesn't run on main but runs on every PR which doesn't make sense. I'm going to set it so it always runs for now since this is required.